code/modules/power/supermatter/supermatter.dm

```diff
@@ -713,7 +713,20 @@
 		return
 	if(moveable && default_unfasten_wrench(user, I, time = 20))
 		return
-	if(user.drop_item())
+	if(istype(I, /obj/item/scalpel/supermatter))
+		var/obj/item/scalpel/supermatter/scalpel = I
+		to_chat(user, "<span class='notice'>You carefully begin to scrape [src] with [I]...</span>")
+		if(I.use_tool(src, user, 10 SECONDS, volume = 100))
+			if(scalpel.usesLeft)
+				to_chat(user, "<span class='danger'>You extract a sliver from [src]. [src] begins to react violently!</span>")
```

Comment:
This has "The supermatter crystal" in it twice, so you can probably change the second one to ", and it begins to react violently!" or something like that.
`"You extract a sliver from the supermatter crystal. The supermatter crystal begins to react violently!"`

Reply:
Will do